Policies/github-candidate-privacy-policy.md

```diff
@@ -0,0 +1,72 @@
+---
+title: GitHub Candidate Privacy Policy
+versions:
+  free-pro-team: '*'
+---
+
+Effective Date: March 8, 2021
+
+This GitHub Candidate Privacy Policy explains what information we collect about you during the application or recruitment process for employment with GitHub, as well as the purposes for which we collect and use that information. 
+
+For the purposes of this policy:
+
+“Candidate” includes applicants and candidates who are part of the GitHub application or recruitment process; and 
+
+"Candidate Personal Information" refers to any information we collect about one of our Candidates during the application or recruitment process which could, alone or together with other information, personally identify them or otherwise be reasonably linked or connected with them. 
+
+This policy does not apply to the use of GitHub services, which are covered under our [GitHub Privacy Statement](/github/site-policy/github-privacy-statement).
+
+### What Candidate Personal Information do we collect?
+
+The Candidate Personal Information we collect, use, and maintain (or “process”) can include the following, but is not limited to:
+
+**Identification and contact information**, including your name, email address, phone number, location, GitHub profile, and any other information you provide (such as referrals and references).
+
+**National identifiers**, such as your citizenship status, residency and work permit status.
+
+**Employment and education history**, such as your resume or CV, cover letter, details of how you heard about the position you are applying for, information about any previous employment at GitHub or affiliates of GitHub, or other information you provide to us in support of an application and/or the application and recruitment process. 
+
+**Professional or employment information**, including your desired salary or terms related to benefits, willingness to relocate, other job preferences, interview details, reference information and/or information received from background checks (where applicable), and information from publicly available resources (such as your LinkedIn profile or website).
+
+**Sensitive or demographic information**, such as your gender, medical or health information, veteran status, or your racial or ethnic origin. 
```

Comment:
I don't think that `gender` (around 1 bit of information) and `racial and ethnic origin` (about 13 bits assumming uniform distribution, but I guess far less) is sensitive. It is usually available for everyone who is able to see person's face. Such as complete strangers on streets.

>name, email address, phone number

are IMHO more sensitive and carry more information.